hkmc2/shared/src/main/scala/hkmc2/codegen/Lifter.scala

```diff
@@ -234,55 +236,38 @@ class Lifter(handlerPaths: Opt[HandlerPaths])(using State, Raise):
     val FreeVars(_, cap) = ctx.usedLocals(f.sym)
 
     val fresh = FreshInt()
-
-    val varsMap = cap.map: s =>
-        val id = fresh.make
-        val nme = s.nme + id + "$"
-        val varSym = VarSymbol(Tree.Ident(nme))
-        val fldSym = BlockMemberSymbol(nme, Nil)
-        val fldDef = TermDefinition(
```

Comment:
@FlandiaYingman turns out we didn't need that crazy dummy term definition at all. And we didn't even need a `ValDefn`, either.